identity_iota/src/did/did.rs

```diff
@@ -31,17 +31,17 @@ pub fn create_address(did: &DID) -> Result<Address> {
 }
 
 /// Creates an 81 Trytes IOTA address from public key bytes for a diff
-pub fn create_diff_address_hash(public_key: &[u8]) -> Result<String> {
-    let hash = &Blake2b256::digest(public_key);
-    let second_hash = &Blake2b256::digest(hash.digest());
-    let encoded: String = encode(second_hash.digest()).into_string();
+pub fn create_diff_address_hash(did: &DID) -> Result<String> {
```

Comment:
And here we should rather use the authentication key instead of the method_id from the DID, because otherwise the address/tag will be the same for every diff, even if we change the authentication key